packages/lesswrong/components/comments/ModerationGuidelinesBox.jsx

```diff
@@ -1,55 +1,125 @@
-import { Components, registerComponent, withDocument} from 'meteor/vulcan:core';
+import { Components, registerComponent, withDocument } from 'meteor/vulcan:core';
 import React, { PureComponent } from 'react';
-import Icon from '@material-ui/core/Icon';
-import { FormattedMessage } from 'meteor/vulcan:i18n';
 import { Posts } from '../../lib/collections/posts';
+import { withStyles } from '@material-ui/core/styles';
+import withNewEvents from '../../lib/events/withNewEvents.jsx';
+import withUser from '../common/withUser';
+import truncatise from 'truncatise';
 import classNames from 'classnames'
-import Users from 'meteor/vulcan:users';
+
+const styles = theme => ({
+  root: {
+    marginBottom: 10,
+    padding: '8px 14px 1px 14px',
+    cursor: 'pointer',
+    '&:hover': {
+      backgroundColor: 'rgba(0,0,0,0.02)'
```

Comment:
I don't like this hover effect. It's right on the border between perceptible and imperceptible, which makes it a bit unsettling when it changes.

Reply:
(This varies based on which monitor I view it on; the three monitors I have in front of me vary quite a bit in how much difference there is between 1.0 and .98 in their tone map.)